### PR TITLE
[DC-2694] update to CDR related notebooks to use service account impersonation

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

**Review by jp3477 — APPROVED** [review]
Looks good, just left a couple questions

File: data_steward/analytics/cdr_ops/rdr_export_qc.py [hunk 1]
```diff
@@ -716,32 +715,6 @@
 query = tpl.render(new_rdr=new_rdr, project_id=project_id)
 execute(client, query)
 
-# # Verify that no joint Personal Family Medical History survey information exists in the Dataset.
```
[the remark this answers is not in the document]
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:719]: That check needs to be dropped.

File: data_steward/analytics/table_metrics/unioned_ehr_scripts/diabetes_completeness.py [hunk 2]
```diff
@@ -13,20 +13,13 @@
 #     name: python3
 # ---
 
-from google.cloud import bigquery
-# %reload_ext google.cloud.bigquery
-from common import PIPELINE_TABLES
```
[the remark this answers is not in the document]
Comment (ksdkalluri) [reply data_steward/analytics/table_metrics/unioned_ehr_scripts/diabetes_completeness.py:18]: that's correct.

[the remark this answers is not in the document]
Comment (jp3477) [reply data_steward/analytics/table_metrics/unioned_ehr_scripts/diabetes_completeness.py:18]: Thanks for answering the questions, this should be good to merge

File: data_steward/analytics/cdr_ops/rdr_export_qc.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:719]: Sorry, @jp3477 was in a meeting earlier and didn't give you the reason for dropping this check, this check needs to be dropped as we are expecting the combined PFMH to be in the RDR export this time. where this check is added to rdr_qc last time is to verify that none of this survey data is in the dataset.

**Pull request merged**